libs/libc/stdlib/Kconfig

```diff
@@ -39,4 +39,12 @@ config LIBC_MAX_TMPFILE
 		maximum size of that last filename.  This size is the size of the full
 		file path.
 
+config LIBC_MAX_EXITFUNS
+	int "Maximum amount of exit functions"
+	default 1 if HAVE_CXX
+	default 0 if !HAVE_CXX
```

Comment:
see my last comment.

Reply:
I still think we need to default to 1 if CXX is defined. The dtors require 1 exit callback function.